build_tools/cmake/build_android_benchmark.sh

```diff
@@ -26,12 +26,12 @@ ninja --version
 python3 --version
 echo "Android NDK path: ${ANDROID_NDK}"
 
+echo "Initializing submodules"
+git submodule update --init
```

Comment:
We want this to be a shallow clone

Reply:
So `git submodule update --jobs 8 --depth 1` everywhere?